src/main/java/org/sagebionetworks/web/client/jsni/Reference.java

```diff
@@ -0,0 +1,33 @@
+package org.sagebionetworks.web.client.jsni;
+
+import com.google.gwt.core.client.JavaScriptObject;
+
+public class Reference extends JavaScriptObject {
```

Comment:
Do we have a standard class naming convention to indicate js objects yet?  Like "ReferenceJsObject"